assets/docs/rishit-dagli/plant-disease/1.md

```diff
@@ -0,0 +1,39 @@
+# Module rishit-dagli/plant-disease/1
+Model to classify among 38 plant disease categories.
+
+<!-- module-type: image-classification -->
+<!-- network-architecture: mobilenet -->
+<!-- dataset: plant_village -->
+<!-- fine-tunable: false -->
+<!-- license: Apache-2.0 -->
+<!-- format: saved_model_2 -->
+<!-- asset-path: https://github.com/Rishit-dagli/Greenathon-Plant-AI/releases/download/v0.1.0/saved_model.tar.gz -->
+
+[![Open Colab notebook]](https://colab.research.google.com/github/Rishit-dagli/Greenathon-Plant-AI/blob/main/notebooks/plant-diseases-training.ipynb)
```

Comment:
Thank you for including this colab link! It's very valuable to be able to see how the model was trained. However, the aim for these colab links at the top of the page is generally to provide a working example of the model that the user can try. In this case, the colab linked here and on your other models can't be run directly because they require various local files to exist.

Could you move these into the description instead? As I said, I think it's valuable information for folks, but it's the wrong place to link it.

Reply:
That definitely makes sense, in fact (if you also see the original repo README), this isn't the original training script but a modified subset that could be tried out on Colab for folks. It makes more sense to not include this Colab. I will later add a Colab with a working example of the model that the user can try.